### PR TITLE
[MFTF] Adding AdminSelectAttributeSetOnEditProductPageActionGroup

### LINKED ISSUE #31290: [Issue] [MFTF] Adding AdminSelectAttributeSetOnEditProductPageActionGroup
This issue is automatically created based on existing pull request: magento/magento2#31287: [MFTF] Adding AdminSelectAttributeSetOnEditProductPageActionGroup

---------
### Description (*)
Adding AdminSelectAttributeSetOnEditProductPageActionGroup

### PR DESCRIPTION
**Pull request by AnnaAPak**
### Description (*)
Adding AdminSelectAttributeSetOnEditProductPageActionGroup

### Resolved issues:
1. [x] resolves magento/magento2#31290: [MFTF] Adding AdminSelectAttributeSetOnEditProductPageActionGroup

### CODE REVIEW

**Comment by AnnaAPak** [comment]
@magento run Functional Tests B2B, Functional Tests EE

**Comment by sidolov** [comment]
@magento create issue

**Comment by AnnaAPak** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

File: app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminSelectAttributeSetOnEditProductPageActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminSelectAttributeSetOnEditProductPageActionGroup">
+        <annotations>
+            <description>Selects the specified value from the Attribute Set dropdown.
+                         The Edit Product Page should be opened prior to Action Group execution</description>
```
Comment (eduard13) [inline app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminSelectAttributeSetOnEditProductPageActionGroup.xml]: I would refactor this part. As for `The Edit Product Page should be opened prior to Action Group execution` sentence, I think that is extra. Usually, you are checking the ActionGroup implementation before using it.

```suggestion
            <description>Selects the specified value from the Attribute Set dropdown on the opened product edit page.</description>
```

**Review by eduard13 — CHANGES_REQUESTED** [review]
Hi @AnnaAPak could you please check the minor comment that I left?
Thank you.

**Comment by AnnaAPak** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

**Comment by AnnaAPak** [comment]
@eduard13  can you please take a look one more time? Thank you.

**Comment by eduard13** [comment]
@magento run all tests

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests B2B, Functional Tests CE

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests B2B, Functional Tests CE

**Review by eduard13 — APPROVED** [review]

**Comment by engcom-Alfa** [comment]
QA not applicable

**Pull request merged**